src/controllers/tx-status.controller.ts

```diff
@@ -33,6 +34,15 @@ export class TxStatusController {
     @param.path.string('txId') txId: string,
   ): Promise<TxStatus> {
     let txStatus:TxStatus;
+    
+    if (!isValidTxId(txId)) {
+      this.logger.warn(`[getTxStatus] the provided tx id: ${txId} is invalid`);
+      txStatus = new TxStatus({
+        type: TxStatusType.INVALID_DATA,
+      });
+      return txStatus;
```

Comment:
Should be a `resolve` call as:
`resolve(txStatus)`
Also please add an unit test in order to check this validation

Reply:
im not sure about why that should be a resolve call, since it has the same structure as the others.

according to unit test: on my way to add it